hathor/simulator/simulator.py

```diff
@@ -250,7 +256,13 @@ def run_until_complete(self,
     def run(self,
             interval: float,
             step: float = DEFAULT_STEP_INTERVAL,
-            status_interval: float = DEFAULT_STATUS_INTERVAL) -> None:
+            status_interval: float = DEFAULT_STATUS_INTERVAL,
+            *,
+            trigger: Optional[Trigger] = None) -> bool:
```

Comment:
What about receiving a list here to support multiple triggers? It could stop if any of them stops it

Reply:
I thought it would be confusing about the stop criteria. It's quite simple to develop a trigger that has a list an apply an OR or AND criteria. What do you think?

Reply:
Ok, you're right, a composed trigger could be implemented if necessary